app/controllers/concerns/hydranorth/batch_edits_controller_behavior.rb

```diff
@@ -11,6 +11,11 @@ def redirect_to_return_controller
         redirect_to sufia.dashboard_files_path
       end
     end
+    #This override is for Sufia 6.2. Issue is fixed in Sufia 7 and this will not be needed.
+    def update_document(obj)
+      super
+      obj.date_modified = Time.current.ctime
```

Comment:
If anybody's curious, this is identical to the corresponding fix in Sufia 7: https://github.com/projecthydra/sufia/blob/master/app/controllers/concerns/sufia/batch_edits_controller_behavior.rb#L31